site-src/geps/gep-713.md

```diff
@@ -38,10 +48,116 @@ portable.
 * Define all potential policies that may be attached to resources
 * Design the full structure and configuration of policies
 
+## Background and concepts
+
+When designing Gateway API, one of the things we’ve found is that we often need to be
+able change the behavior of objects without being able to make changes to the spec
+of those objects. Sometimes, this is because we can’t change the spec of the object
+to hold the information we need ( ReferenceGrant, from GEP-709, affecting Secrets
+and Services is an example), and sometimes it’s because we want the behavior change
+to flow across multiple objects (this is what Policy Attachment is for).
+
+To put this another way, sometimes we need ways to be able to affect how an object
+is interpreted in the API, without representing the description of those effects
+inside the spec of the object.
+
+This document describes the ways we design objects to meet these two use cases,
+and why you might choose one or the other.
+
+We use the term “metaresource” to describe both of these use cases, since the
+object is in both cases a superset of its settings and the settings of a wrapped
+object (or objects). “Meta” here is used in its Greek sense of “more comprehensive”
+or “transcending”, and “resource” rather than “object” because “metaresource”
+is more pronounceable than “metaobject”. Additionally, a single word is better
+than a phrase like “wrapper object” or “wrapper resource” overall, although both
+of those terms are effectively synonymous with “metaresource”.
+
+All ReferenceGrants are metaresources, and all objects that use Policy Attachment
+are metaresources, but metaresources can be more than just those two things.
+
+### Targeted metaresources
+
+ReferenceGrant is an example of a targeted metaresource. It’s tightly bound to a
+particular Kind within a single namespace, and only modifies the behavior of the
+objects that match its binding (by allowing references to them from outside their
+namespace).
+
+We’ve been discussing the idea of having a new object that holds extra capabilities
+for a backend Service (something like BackendCapabilities or similar, name TBD),
```

Comment:
I'm hesitant to make too many references to resources that have not yet been defined. Generally once a GEP is merged, we tend to forget about it - and references to terms like this may soon become outdated. Maybe it would be useful to make this a bit more generic and future proof by focusing on the general concept of extending a Service.

Reply:
I've updated this, PTAL.